ember-element-helper/src/helpers/element.d.ts

```diff
@@ -0,0 +1,11 @@
+import Helper from '@ember/component/helper';
+import type { ComponentReturn } from '@glint/template/-private/integration';
```

Comment:
Relying on private types for this is (a) dangerous, but (b) fortunately unnecessary 🙂

You can do this with public API _and_ actually get the correct typing for it as well by defining the signature as something like:

```ts
export interface ElementHelperSignature<T extends string> {
  Args: {
    Positional: [name: T];
  };
  Return: ComponentLike<{
    Element: 
      T extends keyof HTMLElementTagNameMap
        ? HTMLElementTagNameMap[T]
        : Element;
    Blocks: { default: [] };
  }>;
}

export default class ElementHelper<T extends string> extends Helper<ElementHelperSignature<T>> {}
```